### ISSUE TITLE
Focusrite Scarlet Series

### ISSUE THREAD

**Issue by mriwhh** [bitfocus/companion-module-requests#920]
Hi,
I'd like to control my Focusrite Scrarlet 3rd gen with my Stream Deck. I hope that there are others who also seek for support of Focusrite Audio Interfaces. Those interfaces are very flexible and provide a high sound quality.
I'm willing to support the development regarding requirements, testing and as an agile coach.
Thanks
Clemens

**Comment by SkylerJEhly** [comment]
How do you intend to control it without a LAN port? Can you provide documentation?

**Comment by PeterJanS** [comment]
It doesnt have a LAN port but is usb connected on a networked computer..

**Comment by mriwhh** [comment]
Good point. The Focusrite is connected by USB and there's an app for iPhone and iPad to control it via network :)
The Focusrite can also be controlled using MIDI. I wonder if there's some Network MIDI gateway or interface that could be used.

**Comment by ggrami150** [comment]
I have a Focusrite also and would like to be able to mute separate speaker sets via the Streamdeck. There's a developer who created an app that would allow the Focusrite to be controlled through the MIDI app from the Streamdeck "app store". Seams like it needs to be used in conjunction with MIDI loop. I've tried it but have not had any luck. I would happily purchase his app if I could send it a MIDI message to control the Focusrite but there is no MIDI software equivalent to the version offered by Streamdeck. There's a video on the site you can check out the setup.

https://www.raduvarga.com/software/focusrite-midi-control#forum 

I think most people would be ok programming each button 
as opposed to having companion learn how to set up each individual piece of hardware. The functionality between a MIDI and the Streamdeck is not really the same so my guess is you could only use the Streamdeck to trigger events.

**Comment by ggrami150** [comment]
> How do you intend to control it without a LAN port? Can you provide documentation?

What kind of documentation do you need?
